plugin/src/main/java/io/jenkins/plugins/coverage/metrics/steps/LineCoverageViewModel.java

```diff
@@ -0,0 +1,76 @@
+package io.jenkins.plugins.coverage.metrics.steps;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import edu.hm.hafner.coverage.FileNode;
+import edu.hm.hafner.coverage.Node;
+
+import org.kohsuke.stapler.export.Exported;
+import org.kohsuke.stapler.export.ExportedBean;
+
+import io.jenkins.plugins.coverage.metrics.model.ChangedLinesModel;
+import io.jenkins.plugins.coverage.metrics.model.FileWithChangedLinesCoverageModel;
+import io.jenkins.plugins.coverage.metrics.model.Type;
+
+@ExportedBean
+public class LineCoverageViewModel {
+
+    private final List<FileWithChangedLinesCoverageModel> filesWithChangedLines;
+
+    public LineCoverageViewModel(final Node node) {
+        this.filesWithChangedLines = getFilesWithChangedLines(node);
```

Comment:
The constructor can get the list of FileWithChangedLinesCoverageModel and directly initialize the attribute here so it can be final.